tests/test_pipreqs.py

```diff
@@ -427,6 +429,30 @@ def test_clean_with_imports_to_clean(self):
             data = f.read().lower()
             self.assertTrue(cleaned_module not in data)
 
+    def test_output_requirements(self):
+        """
+        Test --print parameter
+        It should print to stdout the same content as requeriments.txt 
+        """
+
+        capturedOutput = io.StringIO()          
+        sys.stdout = capturedOutput
+
+        pipreqs.init(
+            {'<path>': self.project, '--savepath': None, '--print': True,
+            '--use-local': None, '--force': None, '--proxy': None,
```

Comment:
@fredgrub, the flake8 issues dont seem to have dissapeared even after rebasing. Could you take a look at them? And its weird that it passed the flake8 test while giving a bunch of warnings and erors...

Reply:
I forgot to format the modifications done in this PR. The flake8 was only applied to the other lines.